tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
+{
+    internal static class DatabaseMonitoringPropagator
```

Comment:
Have you thought of having this class implement `IContextInjector` as our others context propagators do? 
It would make this class a tad more complicated, but conform to what we do elsewhere. 🤔 Though I like the simplicity here, so this is maybe better like that :)